recipes/rust/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.18.0" %}
+
+package:
+  name: rust
+  version: {{ version }}
+
+source:
+  fn: rust-{{ version }}.tar.gz
+  url: https://static.rust-lang.org/dist/rustc-{{ version }}-src.tar.gz
+  sha256: d2dc36e99b9e2269488b2bcddde43c234e6bde03edf70cba82a027ff49c36111
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - pkg-config
+    - llvmdev 4.0.*
+    - python 2.7*
+    - openssl 1.0.*
+    - curl 7.52.*
+    - zlib 1.2.8
```

Comment:
You can use jemalloc conda package to save some build time, but that can be done in the feedstock.